indra/newview/llappviewerwin32.cpp

```diff
@@ -722,24 +722,25 @@ bool LLAppViewerWin32::init()
         }
         else
         {
-            Json::Reader reader;
-            Json::Value build_data;
-            if (!reader.parse(inf, build_data, false)) // don't collect comments
+            boost::json::error_code ec;
+            boost::json::value build_data = boost::json::parse(bas, ec);
```

Comment:
Github build throws `llappviewerwin32.cpp(726,64): error C2065: 'bas': undeclared identifier` here. However, my local Windows build is fine.

Reply:
Should be fixed with the next commit!